### ISSUE TITLE
[Feature] Add resampling for image zooming

### ISSUE THREAD

**Issue by cpz3501** [nomacs/nomacs#581]
**Is your feature request related to a problem? Please describe.**
Low resolution / zoom images look bad.

**Describe the solution you'd like**
Option to resample with lanczos.

**Describe alternatives you've considered**
Bicubic or bilinear?

**Additional context**
Not sure why this hasn't been implemented yet, every other image viewer seems to have it. Would be nice if nomac could get it too.

**Comment by diemmarkus** [comment]
did you enable anti-aliasing? (View > Anti-Aliasing)

**Comment by cpz3501** [comment]
Yes, and it only works if you're zooming out or they are in high resolution (you said something similar in #122).

We need filtering for zooming in and low resolution.

**Comment by jackloomen** [comment]
anti-aliasing doesn't appear to be functioning correctly.

when I open a large image and start zooming out, aliasing appears (downsampling is not correct)
you can see it by having smooth lines or curves become dots.

**Comment by scrubbbbs** [comment]
Zooming above 200%, nomacs will use nearest-neighbor scaling with no AA. I think the intent was for working with icons or sprite graphics; you don't see any interpolated pixels.
